usr/sbin/registercloudguest

```diff
@@ -123,10 +123,16 @@ def register_modules(extensions, products, registered=[]):
                 '--product',
                 triplet
             ]
-            if os.path.exists(instance_data_filepath):
+            if os.path.exists(instance_data_filepath) and not args.reg_code:
```

Comment:
We are going to need a change to SUSEConnect. We still want the instance data on the RMT server for which we are the proxy.